### DISCUSSION TITLE
how to make a web game with fresh wasmbuild and bevy ?

### DISCUSSION THREAD

**Discussion by mjcc30** [freshframework/fresh#469] [Q&A]
Hello!

I want to make a [game](https://www.youtube.com/watch?v=j7qHwb7geIM) with [bevy](https://bevyengine.org/)
i saw that i can make a wasm file with 
[wasm-pack](https://github.com/rustwasm/wasm-pack)
or
[wasmbuild](https://github.com/denoland/wasmbuild) from denoland

I really don't know if it's possible to do that with deno or how to do that with fresh and islands

If you have any ideas?

Thanks

**Comment by zicklag** [comment]
I'm not super acquainted with Fresh yet, but running a Bevy game in the browser is really just a matter of serving a WASM file, and a Javascript file, and then running a JavaScript function to start the game.

You don't really need fresh in that case, if you just want to host the game on it's own page where it takes up the whole page.

**Reply by mjcc30** [reply]
yes i know i can make a simple html and with js but i want to use fresh framework and wasm to learn how to use rust in fresh with islands and i don't know how to do this
